frontend/src/app/libs/user-profile-page/user-profile-page-routing.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+
+import { UserProfilePageComponent } from './user-profile-page.component';
+
+const routes: Routes = [{ path: '', component: UserProfilePageComponent }];
```

Comment:
add route path to this page

Reply:
i have lazy loading to this component, is it necessary - to write route path here?